web/pom.xml

```diff
@@ -100,6 +100,18 @@
         </dependency>
 
         <dependency>
+            <groupId>com.fasterxml.jackson.jaxrs</groupId>
```

Comment:
Yeh. I have some ideas on how to do a REST server an and an actor based impl. They are next on my list